www/console_view/src/styles/styles.less

```diff
@@ -1,197 +1,57 @@
+    }
 
-        .fa {
-          cursor: pointer;
-          margin-right: 10px;
+    .row-header {
+        &.has-expanded {
+        min-width: @row-header-width-expanded;
+        max-width: @row-header-width-expanded;
+        width: @row-header-width-expanded;
         }
-      }
+        min-width: @row-header-width;
+        max-width: @row-header-width;
+        width: @row-header-width;
+    }
 
-      .builders {
+    .builder {
         position: relative;
+        float: left;
         height: 100%;
-        background: #ffffff;
-        padding-right: 30px;
-
-        .builder {
-          position: relative;
-          float: left;
-          height: 100%;
-          text-align: center;
-          vertical-align: middle;
-          margin-top: 30px;
-          -webkit-transform: translateX(5px) rotate(-60deg);
-          -moz-transform: translateX(5px) rotate(-60deg);
-          -ms-transform: translateX(5px) rotate(-60deg);
-          -o-transform: translateX(5px) rotate(-60deg);
-          transform: translateX(5px) rotate(-60deg);
-          text-decoration: none;
-          white-space: nowrap;
-        }
-      }
-    }
-  }
-
-  .content {
-    position: relative;
-    width: 100%;
-    height: 100%;
-
-    .change-rows {
-      position: absolute;
-      top: 0;
-      bottom: 110px;
-      overflow-y: auto;
-      overflow-x: visible;
-
-      .change-row {
-        width: 100%;
-        height: 30px;
-        padding-right: 30px;
-        border-top-style: dotted;
-        border-top-width: thin;
-        border-top-color: black;
-        .name {
-          width: 90%;
-          padding-left: 5%;
-          padding-right: 5%;
-          margin: 5px 0 5px 0;
-          white-space: nowrap;
-          overflow: hidden;
-        }
-
-        .fa {
-          margin-right: 10px;
-
-          &.fa-chevron-circle-right {
-            cursor: pointer;
-            transition: 0.5s all ease;
-            -webkit-transition: 0.5s all ease;
-          }
-        }
-
-        .info-collapse {
-          height: initial;
-          padding-left: 35px;
-
-          .panel {
-            margin-top: 15px;
-          }
-        }
-
-        .info {
-          width: 100%;
-
-          .fa {
-            float: left;
-            line-height: 25px;
-          }
-
-          p {
-            padding-left: 22px;
-            white-space: pre-wrap;
-          }
-        }
-
-        .builds {
-          position: relative;
-          height: 100%;
-
-          .build {
-            position: relative;
-            float: left;
-            height: 100%;
-            text-align: center;
-            vertical-align: middle;
-            text-decoration: none;
-
-            .badge-status {
-              cursor: pointer;
-            }
-          }
-        }
-      }
+        text-align: center;
+        vertical-align: middle;
+        margin-top: 30px;
+        -webkit-transform: translateX(0px) rotate(-15deg);
```

Comment:
Are the vendor prefixed versions still needed? It appears everyone has [supported the standard version for quite some time](https://caniuse.com/#feat=transforms2d).